### PR TITLE
Added responsibilities, edited slack, and corrected typos

### PR DESCRIPTION
**Pull request by gerardm27**
## What this PR does / why we need it

Added responsibilities, edited slack, and corrected typos

- [x] I have read the contributing guidelines
- [x] I abide by this repository Code of Conduct

### CODE REVIEW

File: _pages/guide/mentor.md [hunk 1]
```diff
@@ -4,35 +4,50 @@ title: Mentor guide
...
 
 During the event, we expect mentors to:
 
-- Act as helpers, moving from group to group and helping hackers with any questions or roadblocks they have. However, mentors can’t submit a project or contribute in a team’s project in such a way that a project is mainly his/her work. Their task is to guide and help teams to carry on their own projects.
-- Help hackers building their team at the team building event, which takes place at the beginning of the hackathon.
-- Be present most of the time. The hackathon starts at 9.00 P.M on Friday and ends
-  at 9.00 A.M. on Sunday . It is not necessary to assist the whole event, but we
+- **Act as helpers**, moving from group to group and helping hackers with any questions or roadblocks they have. However, keep in mind that mentors can’t submit a project nor contribute in a team’s project in such a way that a project is mainly their work. Their task is to guide and help teams to carry on their own projects.
+
+- Help hackers **building their team** at the team building event, which takes place at the beginning of the hackathon.
```
Comment (jmiarons) [inline _pages/guide/mentor.md:19]: Aixo ho treuria potser perque si volem fer el briefing a la mateix hora que el team building no te massa sentit

Comment (gerardm27) [reply _pages/guide/mentor.md:19]: Oh my bad cert

Comment (gerardm27) [reply _pages/guide/mentor.md]: Stas outdated bro

**Review by jmiarons — APPROVED** [review]

File: _pages/guide/mentor.md (hunk 1 above)
Comment (jmiarons) [reply _pages/guide/mentor.md]: vale bro

**Pull request merged**